alchemy/src/cloudflare/dispatch-namespace.ts

```diff
@@ -0,0 +1,192 @@
+import type { Context } from "../context.js";
+import { Resource } from "../resource.js";
+import { type CloudflareApiOptions, createCloudflareApi } from "./api.js";
+
+/**
+ * Properties for creating or updating a Dispatch Namespace
+ */
+export interface DispatchNamespaceProps extends CloudflareApiOptions {
+  /**
+   * Name of the dispatch namespace
+   * @default id
+   */
+  namespace?: string;
+
+  /**
+   * Whether to adopt an existing namespace with the same name if it exists
+   * If true and a namespace with the same name exists, it will be adopted rather than creating a new one
+   *
+   * @default false
+   */
+  adopt?: boolean;
+}
+
+/**
+ * Output returned after DispatchNamespace creation/update
+ */
+export interface DispatchNamespace
+  extends Resource<"cloudflare::DispatchNamespace">,
+    DispatchNamespaceProps {
+  /**
+   * Resource type identifier
+   */
+  type: "dispatch_namespace";
+
+  /**
+   * The ID of the dispatch namespace
+   */
+  id: string;
+
+  /**
+   * The name of the dispatch namespace
+   */
+  namespace: string;
+
+  /**
+   * Time at which the namespace was created
+   */
+  createdAt: number;
+
+  /**
+   * Time at which the namespace was last updated
+   */
+  updatedAt: number;
+}
+
+/**
+ * A Cloudflare Workers for Platforms Dispatch Namespace enables deploying user workers
+ * that can be dynamically invoked through a dispatcher worker.
+ *
+ * @example
+ * // Create a basic dispatch namespace for user workers:
+ * const userNamespace = await DispatchNamespace("user-workers", {
+ *   namespace: "user-workers"
+ * });
+ *
+ * @example
+ * // Create a dispatcher worker that can route to user workers:
+ * const dispatcher = await Worker("dispatcher", {
+ *   entrypoint: "./src/dispatcher.ts",
+ *   bindings: {
+ *     NAMESPACE: userNamespace
+ *   }
+ * });
+ *
+ * @example
+ * // Deploy a user worker to the dispatch namespace:
+ * const userWorker = await Worker("user-worker", {
+ *   entrypoint: "./src/user-worker.ts",
+ *   dispatchNamespace: userNamespace
+ * });
+ *
+ * @see https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/get-started/user-workers/
+ */
+export const DispatchNamespace = Resource(
```

Comment:
Make sure to create the alchemy-web/providers/cloudflare/dispatch-namespace.md. follow pattern of existing docs in there and include each of the examples